README.md

```diff
@@ -75,7 +75,7 @@ snakemake -j 1 -p --configfile config/config_mpxv.yaml
 snakemake -j 1 -p --configfile config/config_hmpxv1.yaml
 ```
 
```

Comment:
I think it'd be appropriate in this PR to also change:

https://github.com/nextstrain/monkeypox/blob/e1631e5e15450b7355636a8c7dcaffe8259baff2/README.md?plain=1#L65-L66

to remove the 2nd line, since Nextclade v2 would be installed by default through all installation paths.

Reply:
Done!